lib/common/utils.ts

```diff
@@ -280,9 +280,15 @@ export function makeZoneAwareRemoveAllListeners(fnName: string, useCapturingPara
   const defaultUseCapturing = useCapturingParam ? false : undefined;
 
   return function zoneAwareRemoveAllListener(self: any, args: any[]) {
+    var target = self || _global;
+    if (args.length === 0) {
+      // remove all listeners without eventName
+      target[EVENT_TASKS] = [];
+      target[symbol]();
+      return;
```

Comment:
The real `removeAllListeners(...)`
return `this`
This wrapper should do the same
https://github.com/nodejs/node/blob/master/lib/events.js#L404